nomad/grapl-core.nomad

```diff
@@ -123,6 +124,26 @@ variable "plugin_registry_kernel_artifact_url" {
   description = "S3 URL specifying the kernel for the Firecracker VM"
 }
 
+variable "organization_management_db_hostname" {
+  type        = string
+  description = "What is the host for the organization management table?"
+}
+
+variable "organization_management_db_port" {
+  type        = string
+  description = "What is the port for the organization management table?"
+}
+
+variable "organization_management_db_username" {
+  type        = string
+  description = "What is the username for the organization management table?"
+}
+
+variable "organization_management_db_password" {
+  type        = string
+  description = "What is the password for the organization management table?"
+}
```

Comment:
These descriptions should be tweaked to refer to a "database" rather than a "table".